src/attributes/codegen.md

```diff
@@ -230,6 +230,27 @@ Feature        | Implicitly Enables | Feature Name
 `tme`          |                | FEAT_TME --- Transactional Memory Extension
 `vh`           |                | FEAT_VHE --- Virtualization Host Extensions
 
+#### `loongarch`
+
+r[attributes.codegen.target_feature.loongarch]
+
+This platform requires that `#[target_feature]` is only applied to [`unsafe`
+functions][unsafe function].
+
+Further documentation on these features can be found in the [LoongArch Reference
+Manual], or elsewhere on [LoongArch GitHub Account].
+
+[LoongArch Reference Manual]: https://github.com/loongson/LoongArch-Documentation
+[LoongArch GitHub Account]: https://github.com/loongson
+
+Feature     | Implicitly Enables  | Description
+------------|---------------------|-------------------
+`frecipe`   |                     | FRECIPE --- Reciprocal approximation instrutions
+`lasx`      | `lsx`               | LASX --- 256-bit vector instructions
+`lbt`       |                     | LBT --- Binary translation instructions
+`lsx`       | `d`                 | LSX --- 128-bit vector instructions
```

Comment:
This mentions `d` as an implicitly enabled feature, but I don't see that as a feature that exists.

Reply:
Sorry for the delayed response due to the holidays. It looks like we are not yet able to stabilize the `lsx` feature, as it implicitly depends on the `d` feature. https://github.com/rust-lang/rust/pull/135015#issuecomment-2567489339